### ISSUE TITLE
[FR]: Go home - keep blades on

### ISSUE THREAD

**Issue by tontsa74** [MTrab/landroid_cloud#631]
### Describe the feature you wish to make a request for

Keep blades on while returning to base. 

### Describe alternatives you've considered

Landroid app have this feature. It's activated with long press of 'Go Home' button. 

### Additional context

https://www.reddit.com/r/worxlandroid/s/E2yz9wpkrR

**Comment by MTrab** [comment]
It should already do this, if the mower supports this feature

**Comment by tontsa74** [comment]
> It should already do this, if the mower supports this feature

Ok, thank you. Should this be a bug issue then? 
I have
Device info
Landroid M700 (WR167E)
Firmware: 3.32.0+1

Keep blades on feature is working via landroid app but not when clicking HA device button or with automation call. Should it just keep blades on every time when returning to base?

**Comment by tontsa74** [comment]
If there is some raw code json to try etc.  to help debugging, please let me know. :)

**Comment by MTrab** [comment]
No, there isn't anything you can do, as it's "behind the scenes" code that I need to debug. 
Not sure when I'll have the time, as it's vacation soon and also I'm moving so my developer environment is currently located in a moving box

**Comment by tontsa74** [comment]
Ok, take your time and have a good vacation 😊. This would be nice because currently I have problems with normal edge-cut but I can live with it.
Thank you for this handy integration! 🙏

**Comment by tontsa74** [comment]
I noticed that when I use scheduling and landroid battery is not under 10% then blades are keeped on while returning to base. This is a good workaround for my problem.
